src/quickstart/install-cli-tools.md

```diff
@@ -25,6 +30,16 @@ Install the latest `helmc` cli for Linux or Mac OS X with:
 
     $ curl -sSL https://get.helm.sh | bash
 
+> **NOTICE:** The `unzip` package is a requirement for this command
+
+The installer places the `helmc` binary in your current directory, but you
+should move it somewhere in your $PATH:
+
+    $ ln -fs $PWD/helmc /usr/local/bin/helmc
+
+> **NOTE:** If you are using Linux you may have to prepend this command
```

Comment:
yeah, use the annotated `!!! note` style as seen [here](https://raw.githubusercontent.com/deis/workflow/master/src/applications/using-docker-images.md) (rendered view [here](https://deis.com/docs/workflow/applications/using-docker-images/)).

EDIT: looks like this is broken so I'll file an issue